src/puppetlabs/trapperkeeper/services/authorization/authorization_service.clj

```diff
@@ -0,0 +1,23 @@
+(ns puppetlabs.trapperkeeper.services.authorization.authorization-service
+  (:require [puppetlabs.trapperkeeper.core :refer [defservice]]
+            [puppetlabs.trapperkeeper.authorization.ring-middleware :refer
+             [wrap-authorization-check]]
+            [puppetlabs.trapperkeeper.services.authorization.authorization-core :refer :all]))
+
+(defprotocol AuthorizationService
+  (wrap-handler [this handler]))
+
+(defservice authorization-service
+  AuthorizationService
+  [[:ConfigService get-in-config]]
+  (init
+    [this context]
+    (let [config (get-in-config [:authorization :rules])]
+      (validate-auth-config! config)
+      (assoc context :authorization {:rules (transform-config config)})))
+
+  (wrap-handler
+    [this handler]
+    (let [context (service-context this)
```

Comment:
Nevermind, a community member filed an issue and there's the answer at https://github.com/puppetlabs/trapperkeeper/issues/166